pkg/config/config.go

```diff
@@ -107,6 +107,7 @@ type FeatureStates struct {
 	VMClassAsConfigDayNDate    bool // FSS_WCP_VM_CLASS_AS_CONFIG_DAYNDATE
 	VMOpV1Alpha2               bool // FSS_WCP_VMSERVICE_V1ALPHA2
 	PodVMOnStretchedSupervisor bool // FSS_PODVMONSTRETCHEDSUPERVISOR
+	TKGMultipleCL              bool // FSS_WCP_TKG_Multiple_CL
```

Comment:
The FeatureState field name does not need to be 1-to-1 with the env var. I prefer it to be more descriptive. Is this essentially `MultipleClusterContentLibrariesForTKRs`?

Reply:
The naming convention we have used for the FSS everywhere is FSS_WCP_TKG_Multiple_CL.
That's why I named it as TKGMultipleCL.
Please let me know if you find something like `TKGMultipleContentLibrary` as more relevant.